.github/workflows/goreleaser.yml

```diff
@@ -176,6 +176,10 @@ jobs:
         with:
           app-id: ${{ secrets.MONDOO_MERGEBOT_APP_ID }}
           private-key: ${{ secrets.MONDOO_MERGEBOT_APP_PRIVATE_KEY }}
+          owner: ${{ github.repository_owner }}
+          repository: |
+            mondoohq/installer
+            mondoohq/cnspec
```

Comment:
This should fix: https://github.com/mondoohq/cnspec/actions/runs/13013356920/job/36296810008

Reference:
https://github.com/orgs/community/discussions/69154
https://github.com/marketplace/actions/create-github-app-token#create-a-token-for-multiple-repositories-in-the-current-owners-installation